command/server/server.go

```diff
@@ -24,6 +21,8 @@ func GetCommand() *cobra.Command {
 	helper.RegisterLegacyGRPCAddressFlag(serverCmd)
 	helper.RegisterJSONRPCFlag(serverCmd)
 
+	// register subcommand used to generate config file on the fly
+	serverCmd.AddCommand(getGenerateConfigCmd())
```

Comment:
This should be called from a special method. Please see how this is done with other commands 🙏

Reply:
Refactored 9350d7d3c963f0b39070691337bd8610bbf3da8a